install.R

```diff
@@ -1,11 +1,7 @@
-install.packages("tidyverse")
-install.packages("rmarkdown")
-install.packages("httr")
-install.packages("shinydashboard")
-install.packages("leaflet")
-install.packages("knitr")
 install.packages("ggplot2")
 install.packages("dplyr")
-install.packages("rlibs")
-install.packages("caTools")
-install.packages("bitops")
+install.packages("lavaan")
+install.packages("plyr")
+install.packages("cowplot")
+install.packages("rmarkdown")
+install.packages("readr")
```

Comment:
THANK YOU for figuring out what's needed! I was flying blind not being able to understand the R output!! 😬

Reply:
When I click on the knitr button in RStudio it tells me that I need to install `caTools` and `bitops`....any chance we can bring them back into the fold?

Reply:
Ah oops, didn't realized those are needed for knitting.